src/clusterfuzz/_internal/base/tasks.py

```diff
@@ -747,6 +747,12 @@ def redo_testcase(testcase, tasks, user_email):
         wait_time=wait_time)
 
   if progression:
+    # Log the task for debug purposes
+    testcase.comments += '[%s] : Redo task: Adding testcase id %s task \
```

Comment:
Can you please just call logs.log_info instead? I think adding it to the comment will annoy users and will only be useful in the very short run.
Also can you hoist this up so it's done on every redo task, not just progression.

Reply:
Done